randovania/games/samus_returns/logic_database/Surface - East.txt

```diff
@@ -136,9 +136,13 @@ Extra - asset_id: collision_camera_003
       All of the following:
           Morph Ball
           Any of the following:
-              Super Jump (Advanced) or Unmorph Extend (Intermediate) or Climb Rooms Vertically (High Jump)
-              # Freeze Enemy
-              Stand on Frozen Enemy (Beginner) and Fully Freeze Enemy
+              Super Jump (Beginner) or Climb Rooms Vertically (High Jump)
```

Comment:
this is not possible to do with unmorph extend?

Reply:
I think so but I gave up trying because a super jump is easier. Could add it back later